index.html

```diff
@@ -93,5 +95,11 @@
            some of the bootstrap css -->
       <link rel="stylesheet" type="text/css" media="all"
             href="style.css" />
+            <div class="figure">
+    <figure>
+           <img src="http://whc.unesco.org/uploads/thumbs/site_1513_0008-375-500-20160616135234.jpg"
```

Comment:
Can you add a caption (given that you're using hte advanced `<figure>` tag anyway?